install.sh

```diff
@@ -11,10 +11,13 @@ if [[ -z "${INSTALLED[*]}" ]]; then
     echo "* creating dkms.conf"
     sed 's/"@DO_NOT_CHANGE@"/"'"${VERSION}"'"/g' <hid-xpadneo/dkms.conf.in >hid-xpadneo/dkms.conf
 
+    echo "* installing sources (using DKMS)"
     # TODO: Works around https://github.com/dell/dkms/issues/177 for DKMS 3
-    echo "* adding hid-xpadneo-${VERSION} folder to /usr/src"
-    mkdir -p "/usr/src/hid-xpadneo-${VERSION}"
-    cp --recursive "${V[@]}" hid-xpadneo/. "/usr/src/hid-xpadneo-${VERSION}/."
+    cp -a hid-xpadneo/ /usr/src/hid-xpadneo-${VERSION}
+    dkms add -v "hid-xpadneo/${VERSION}" --force || cat_dkms_make_log
```

Comment:
Looking at the build logs, the parameter `-v` seems to be conflicting. I'm not sure why you added it.